File: root/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Logic/Common/TestReturnValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

// ベースクラス
using Touryo.Infrastructure.Business.Common;

namespace MVC_Sample.Logic.Common
{
    public class TestReturnValue : MyReturnValue
    {
        /// <summary>汎用エリア</summary>
        public object Obj;

        /// <summary>ShipperID</summary>
        public int ShipperID;

        /// <summary>CompanyName</summary>
        public string CompanyName;

        /// <summary>Phone</summary>
        public string Phone;
    }
}

File: root/programs/VB/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Logic/Common/TestReturnValue.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

' ベースクラス
Imports Touryo.Infrastructure.Business.Common

Namespace Logic.Common
    Public Class TestReturnValue
        Inherits MyReturnValue
        ''' <summary>汎用エリア</summary>
        Public Obj As Object

        ''' <summary>ShipperID</summary>
        Public ShipperID As Integer

        ''' <summary>CompanyName</summary>
        Public CompanyName As String

        ''' <summary>Phone</summary>
        Public Phone As String
    End Class
End Namespace
